pkg/gatewayserver/config.go

```diff
@@ -66,6 +66,8 @@ type Config struct {
 	MQTTV2       config.MQTT        `name:"mqtt-v2"`
 	UDP          UDPConfig          `name:"udp"`
 	BasicStation BasicStationConfig `name:"basic-station"`
+
+	DisconnectTTL time.Duration `name:"disconnect-ttl"`
```

Comment:
I would keep this under `UpdateConnectionStatsDebounceTime`, and I think a better name would be `ConnectionStatsDisconnectTTL`. You should also add a `description`.